src/app/api/chapter-request/route.schema.ts

```diff
@@ -0,0 +1,84 @@
+import { z } from "zod";
+
+/*
+  firstName            String
+  lastName             String
+  universityEmail      String    @unique
+  phoneNumber          String
+  university           String
+  universityAddress    String
+  leadershipExperience String
+  motivation           String
+  availabilities       String
+  questions            String
+*/
+export const ChapterRequest = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  universityEmail: z.string().email("This is not a valid email"),
+  phoneNumber: z.string(),
+  university: z.string(),
+  universityAddress: z.string(),
+  leadershipExperience: z.string(),
+  motivation: z.string(),
+  availabilities: z.string(),
```

Comment:
I would leave a todo here. Passing a list of time availabilities as a string seems a bit counterintuitive. But we also don't fully know how the frontend might choose to parse the time.